### DISCUSSION TITLE
Facepoke inside Krita AI

### DISCUSSION THREAD

**Discussion by destromestro** [Acly/krita-ai-diffusion#1301] [Ideas]
Hi coding gurus,
It would be amazing to have integrate Facepoke in to Krita AI if possible. Then we can select an area with a face and then change the expression and add the change as a layer. Maybe with some generative blending. I wish if I had the knowledge to code something like that. Super easy when you create comics and consistence scenes with differenct expressions. 
Someone....please do this..

**Comment by minsky91** [comment]
This appeal should be done to the ComfUI developer community. If someone coded a Comfy node for Facepoke, having it working in Krita AI would be only a matter of days.

**Reply by destromestro** [reply]
Lookin in to this, I found this node. This is excatly what you are looking for. 

https://github.com/PowerHouseMan/ComfyUI-AdvancedLivePortrait

Not sure how to get the attention of @Acly (The Krita AI GOD, I have immense respect for him). An in-built feature to select a face of an already generated image and change the expression and add it as an additional layer would be exptremely valuable for storyboarding and many other things.

**Reply by minsky91** [reply]
Ah, that's a good news! Now that such a node exists, it shouldn't be too difficult to adapt it for using in the Krita AI plugin: the latest version has support for custom workflows. Come and join Krita AI Discord, there are knowledgeable people who have already created Krita-adapted workflows, and a special channel for sharing them. Ask those people or learn this technique yourself, you don't need Acly for that.

**Reply by destromestro** [reply]
Oh.. is there a Discord? I can't find it. Can you kindly post a link to the Discord?

**Reply by minsky91** [reply]
it's 
https://discord.com/invite/pWyzHfHHhU
Enjoy!
